ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -254,6 +254,36 @@ EOF
     ansible-playbook -i "${SHARED_DIR}/ansible-hosts" /tmp/repo.yaml -vvv
 }
 
+# Do sdn migration to ovn since sdn is not supported from 4.17 version
+function sdn2ovn(){
+    TARGET=${TARGET:-OVNKubernetes}
```

Comment:
```suggestion
    local target_network_plugin=${target_network_plugin:-OVNKubernetes}
```
`TARGET` is already used in other code, to avoid conflicts, suggest to use a different var name.

Reply:
ah, good catch.  Sounds like `TARGET` is not used for this function,  I removed it. thanks.